### PR TITLE
Fix warning for plugins that override this method with the specified …

### PR DESCRIPTION
**Pull request by bor0**
…parameter

### CODE REVIEW

**Comment by bor0** [comment]
Actually I'm closing this, as the current code doesn't make much sense.
And if we do `get_page_base_url($deprecated = 0)` or `get_page_base_url($deprecated = null)` the same warning is generated.

**Comment by bor0** [comment]
On a second thought, I think we can deprecate the function as a whole to make things easier on plugin devs.

**Comment by bor0** [comment]
After further discussion with @mikejolley, it doesn't make sense to deprecate this method since it was introduced in 3.3 1b22d97be292a1d826e4c4c5a725d4952029b6ca. We can rename it instead for an easier fix on potential existing overrides.

File: includes/abstracts/abstract-wc-widget.php [hunk 1]
```diff
@@ -280,7 +280,7 @@ public function form( $instance ) {
 	 *
 	 * @return string
```
Comment (mikejolley) [inline includes/abstracts/abstract-wc-widget.php]: Add since 3.3 tag

**Review by mikejolley — APPROVED** [review]
LGTM just needs a small addition.

**Pull request merged**